app/assets/javascripts/practice_editor_overview.es6

```diff
@@ -7,25 +7,30 @@
     const TAGLINE_CHARACTER_COUNT = 150;
     const SUMMARY_CHARACTER_COUNT = 400;
 
+    const facilityOption = '#initiating_facility_type_facility';
+    const visnOption = '#initiating_facility_type_visn';
+    const departmentOption = '#initiating_facility_type_department';
+    const otherOption = '#initiating_facility_type_other';
+
     function countCharsOnPageLoad() {
         let practiceNameCurrentLength = $('.practice-editor-name-input').val().length;
-        let practiceTaglineCurrentLength = $('.practice-editor-tagline-textarea').val().length;
+        // let practiceTaglineCurrentLength = $('.practice-editor-tagline-textarea').val().length;
```

Comment:
nit: can we remove the commented out code in this file?

Reply:
Yes, for now. I guess we don't need it since we aren't doing character counts lol.